engine/crates/engine-v2/Cargo.toml

```diff
@@ -31,6 +30,7 @@ thiserror.workspace = true
 futures-util.workspace = true
 hex = "0.4.3"
 url.workspace = true
+tracing.workspace = true
```

Comment:
Should we be using `log` since `tracing` doesn't work on production?

Reply:
For now, I'm only using those with `DEBUG` for stuff we would want to log in production imho and tracing provides nice formatting in tests. Kinda tempted to keep that as is for now to be honest...

Reply:
Assuming you mean stuff we would not want to log in production then ok

Reply:
yeah, missed the "not" 😅